src/lib/viewers/archive/ArchiveExplorer.js

```diff
@@ -115,53 +118,64 @@ class ArchiveExplorer extends React.Component {
         });
     };
 
+    /**
+     * Handle click event, update fullPath state
+     *
+     * @param {string} fullPath - target folder path
+     * @return {void}
+     */
+    handleClickFullPath = fullPath => this.setState({ fullPath });
+
     /**
      * render data
      *
      * @return {jsx} VirtualizedTable
      */
     render() {
         const { itemCollection } = this.props;
-        const { fullPath } = this.state;
+        const { fullPath, view } = this.state;
         const itemList = this.getItemList(itemCollection, fullPath);
 
         return (
             <Internationalize language="en-us" messages={{}}>
-                <VirtualizedTable
-                    className="ArchiveFilesTable"
-                    rowData={itemList}
-                    rowGetter={this.getRowData(itemList)}
-                >
-                    {intl => [
-                        <Column
-                            key={KEY_NAME}
-                            cellRenderer={itemNameCellRenderer(intl, this.handleClick)}
-                            dataKey={KEY_NAME}
-                            disableSort
-                            flexGrow={3}
-                            label={__('filename')}
-                            width={1}
-                        />,
-                        <Column
-                            key={KEY_MODIFIED_AT}
-                            cellRenderer={readableTimeCellRenderer}
-                            dataKey={KEY_MODIFIED_AT}
-                            disableSort
-                            flexGrow={2}
-                            label={__('last_modified_date')}
-                            width={1}
-                        />,
-                        <Column
-                            key={KEY_SIZE}
-                            cellRenderer={sizeCellRenderer()}
-                            dataKey={KEY_SIZE}
-                            disableSort
-                            flexGrow={1}
-                            label={__('size')}
-                            width={1}
-                        />,
-                    ]}
-                </VirtualizedTable>
+                <div className="bp-ArchiveExplorer">
+                    <Breadcrumbs fullPath={fullPath} onClick={this.handleClickFullPath} view={view} />
+                    <VirtualizedTable
+                        className="ArchiveFilesTable"
```

Comment:
We'll want to continue prefixing our classes with `bp-` to avoid any conflicts with consuming apps.